FAUCETS.md

```diff
@@ -0,0 +1,29 @@
+# Testnet Token Faucets
+
+Rococo:
+
+- https://github.com/paritytech/cumulus/blob/master/parachains/runtimes/contracts/contracts-rococo/README.md#rococo-deployment
+
+- https://app.element.io/#/room/#rococo-faucet:matrix.org
```

Comment:
You could also add web version of it if it's relevant https://paritytech.github.io/polkadot-testnet-faucet/